Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py

```
@@ -0,0 +1,1961 @@
+                    )
+                time.sleep(EVENTS_INTERVAL_SECS)
+        except Exception:
+            failures += 1
+    return []
+
+
+def try_create_search_with_retry(client, events_query, offense, max_retries=None):
+    if not max_retries:
+        max_retries = EVENTS_FAILURE_LIMIT
+    failures = 0
+    search_created_successfully = False
+    query_status = ""
+    search_id = ""
+    while not search_created_successfully and failures <= max_retries:
+        try:
+            raw_search = client.search(events_query)
+            search_res_events = deepcopy(raw_search)
+            search_res_events = filter_dict_non_intersection_key_to_value(
+                replace_keys(search_res_events, SEARCH_ID_NAMES_MAP),
+                SEARCH_ID_NAMES_MAP,
+            )
+            search_id = search_res_events.get("ID")
+            query_status = search_res_events.get("Status")
+            search_created_successfully = True
+        except Exception:
+            failures += 1
+    if failures >= EVENTS_FAILURE_LIMIT:
+        raise DemistoException(f"Unable to create search for offense: {offense['id']}.")
+    return query_status, search_id
+
+
+def fetch_raw_offenses(client: QRadarClient, offense_id, user_query):
+    """
+    Use filter frames based on id ranges: "id>offense_id AND id<(offense_id+incidents_per_fetch)"
+
+    If couldn’t fetch offense:
+        Fetch last fetchable offense, and set it as the upper limit
+        is limit greater than last fetched offense id?
+             yes - fetch with increments until manage to fetch (or until limit is reached - dead condition)
+             no  - finish fetch-incidents
+    """
+    # try to adjust start_offense_id to user_query start offense id
+    try:
+        if "id>" in user_query:
+            user_offense_id = int(user_query.split("id>")[1].split(" ")[0])
+            if user_offense_id > offense_id:
+                offense_id = user_offense_id
+    except Exception:
+        pass
```

Why do you swallow an exception? It is later very hard to debug